.github/workflows/deseq_workshop_1-spring-tag.yml

```diff
@@ -0,0 +1,53 @@
+name: Tag production image for deseq_workshop_1
+on: [workflow_dispatch]
+
+env:
+  CLASS: deseq_workshop_1
```

Comment:
In the spirit of the conversation on slack, change this to `deseq` and also the file name

Reply:
Just be careful to change all instance in the files